README.md

```diff
@@ -10,34 +10,33 @@ For tutorials, samples, quick starts, and other documentation, go to [Azure for
 ## Packages available
 Each service might have a number of libraries available from each of the following categories discussed below:
 
-* [Client - July 2019 Preview](#Client-July-2019-Preview)
-* [Client - Stable](#Client-Stable)
-* [Management](#Management)
+- [Client - November 2019 Releases](#Client-November-2019-Releases)
+- [Client - Previous Versions](#Client-Previous-Versions)
+- [Management](#Management)
```

Comment:
DO people understand what client and management even mean? If not, maybe it would be good to define the terms (or add a footnote)

Reply:
100% agreed - it is defined in the management section once people click on it or scroll (we can ditch this menu but I do not think it would add clarity)